libsrc/atari2600/crt0.s

```diff
@@ -0,0 +1,46 @@
+; Atari VCS 2600 startup code for cc65
+;
+; Florent Flament (contact@florentflament.com), 2017
+
+        .export         _exit
+        .export         __STARTUP__ : absolute = 1
+
+        .import         __RAM_START__, __RAM_SIZE__
+        .import         copydata
+        .import         _main
+
+        .include        "zeropage.inc"
+
+
+.segment "STARTUP"
+start:
+; Clear decimal mode
+        cld
+; Clear zeropage & registers
+; Also Initializes system stack pointer
+        ldx     #0
```

Comment:
Though, we might want to sacrifice 1 byte in favour of more clarity. What would you prefer ?